src/App.js

```diff
@@ -49,9 +54,9 @@ class App extends Component {
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
-
-               // Additional props (event callbacks, etc.) can be passed here
-             />;
+               type={item.type} // add item type to what's being returned
+               onItemClicked={this.onItemClicked} // connect item to points
```

Comment:
Make sure that you are highlighting what the purpose of that function is with its name. If it’s a callback, use the term callback, if it’s an event handler, tell us what kind of event it is.

For instance `increaseScoreCallback = {this.increasescore}`. Save the `onItemClicked` name for the event handler function in GameItem.js